jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/PostTimestampsIT.java

```diff
@@ -710,8 +736,8 @@ public void testLogicForVoyageNumberFields() {
       .assertThat()
       .statusCode(400);
 
-    // remove exportVoyageNumber to test that when(exportVoyageNumber & importVoyageNumber) are not given together.
-    map.remove("exportVoyageNumber");
+    // remove carrierExportVoyageNumber to test that when(carrierExportVoyageNumber & carrierImportVoyageNumber) are not given together.
+    map.remove("carrierExportVoyageNumber");
```

Comment:
This is still a valid timestamp1_1 (line 727), so nothing is being removed in the map. 

I think we should keep this test, as part of backwards comp for JIT 1.1